explorer/app/components/Project/project.tsx

```diff
@@ -34,12 +35,14 @@ export const Project = ({
   const ProjectOverviewMain = tablet ? Main : Fragment;
   const ProjectOverviewSide = tablet ? Side : Fragment;
   return (
-    <ProjectLayout>
+    <Container>
```

Comment:
Is `<Container>` still required here? If not, would you be able to revert the component hierarchy to its original format?